documents-webapp/src/main/webapp/vue-app/documents/components/header/actions/DocumentsAddNewFile.vue

```diff
@@ -8,7 +8,7 @@
 export default {
   computed: {
     isMobile() {
-      return this.$vuetify.breakpoint.name === 'xs';
+      return this.$vuetify.breakpoint.name === 'xs' || this.$vuetify.breakpoint.name === 'sm';
```

Comment:
Isn't 'sm' breakpoint for tablets ? do we have the same design for tablets and mobile

Reply:
Yes its the same for mobile and tablet